{{cookiecutter.github_project_name}}/js/package.json

```
@@ -22,6 +22,8 @@
   "scripts": {
     "clean": "rimraf dist/",
```

```suggestion
    "clean": "rimraf dist/ && rimraf ../{{ cookiecutter.python_package_name }}/static",
```

This ensures that the static file directory is also cleaned up.